src/argilla/client/apis/datasets.py

```diff
@@ -181,8 +183,13 @@ def scan(
         if limit == 0:
             limit = None
 
+        fields_to_load = ["*"]
+        if fields:
+            fields.extend(["id", "inputs"])
+            fields_to_load = list(set(fields))
+
         request = {
-            "fields": list(projection) if projection else ["id"],
+            "fields": fields_to_load,
```

Comment:
```suggestion
            "fields": list(projection) if projection else ["id"],
```

Reply:
```
else ["id"]
```
should be `["id", "inputs"]`.

This is bug as the object verification expecting at least id and input. See my comment here https://github.com/argilla-io/argilla/issues/2398#issuecomment-1444482828